Doc/configuration.md

```diff
@@ -302,6 +306,21 @@ This is used for legacy purposes, when you work with specific event classes.
 
 If you don't need it, use the null value: `~`.
 
+### Tag resolvers
+
+2 services are injected into tag value resolution: 
+- `container` = container
+- `request` = current request. This is an alias for `container.get('request_stack').getCurrentRequest()`
+
+To activate those resolvers, you need to add: `@=` at the beggining of you tag value:
+```yaml
+tagName: '@=request.get("X-Header")'
```

Comment:
How can we put a default value. For exemple, there is no request when executing a command ;)

Reply:
@valentin-claras: I added some examples in the documentation. Is it ok for you?